modules/Movecraft/src/main/java/net/countercraft/movecraft/mapUpdater/update/ExplosionUpdateCommand.java

```diff
@@ -33,7 +32,11 @@ public float getStrength() {
     public void doUpdate() {
         //if (explosionStrength > 0) { // don't bother with tiny explosions
         //Location loc = new Lo cation(explosionLocation.getWorld(), explosionLocation.getX() + 0.5, explosionLocation.getY() + 0.5, explosionLocation.getZ());
-        if (Settings.Debug){
+        ExplosionEvent e = new ExplosionEvent(explosionLocation, explosionStrength);
```

Comment:
why are we creating a custom event over a preexisting one such as `BlockExplodeEvent`

Reply:
`BlockExplodeEvent` is for after an explosion has been created and there is a list of blocks which can be destroyed.  This is called before Movecraft creates an explosion.